ee/fe/modules/340-extended-monitoring/monitoring/prometheus-rules/certificates/certificate.yaml

```diff
@@ -0,0 +1,62 @@
+- name: kubernetes.certmanager.certificate
+  rules:
+  - alert: CertificateSecretExpiredSoon
+    expr: |
+      max by (name, namespace) (
+        cert_exporter_secret_not_after{job="cert-exporter", secretkey!="ca.crt"} - time() < 1209600
+      ) * on (namespace) group_left() max by (namespace) (extended_monitoring_enabled)
+    for: 1h
+    labels:
+      severity_level: "8"
+    annotations:
+      plk_protocol_version: "1"
+      plk_incident_initial_status: "todo"
+      plk_markup_format: "markdown"
+      plk_pending_until_firing_for: "30m"
+      plk_grouped_by__main: "CertificateSecretExpiration,tier=cluster,prometheus=deckhouse"
```

Comment:
```suggestion
      plk_grouped_by__main: "CertificateSecretExpiration,tier=~tier,prometheus=deckhouse"
```

Reply:
This is needed to groups alerts by tiers to route them. Alerts about certificates from system namespaces will have the `tier-cluster` label, and from other namespaces will have the `tier=application`.